### ISSUE TITLE
Add possibility for interpreting both most positive and most negative terms for topics

### ISSUE THREAD

**Issue by x-tabdeveloping** [x-tabdeveloping/turftopic#29]
### Rationale:
S3 interprets topics as axes of semantics. Whether terms rank positively on a feature or negatively is basically arbitrary, and both ends of the spectrum should in theory convey meaningful information.
As such we should recommend users in the paper to use both ends for interpreting what the axes mean and provide utilities in the package that aid this interpretative work.

### Proposal:
Implement printing and exporting utilities that support this kid of behaviour: 

For instance:
```python
model.print_topics(display_negative=True)
```
| Topic ID         | Positive     | Negative |
|--------------|-----------|------------|
| 0 | Dawg, Darg, Hot Diggity Dog, ...      | Cold Non-dog, Cool Catty-Cat, ... |
| 1      |  Gooning, Edging, Henchmen, Spooning, ...  | Girlbossing, Gatekeeping, CEO, ... |

We should potentially rename `export/print_highest_ranking_documents` to `export/print_representative_documents` and have a `display_negative` flag.

Additionally we should probably make it so that the default behaviour of S3 is this, so that users do not have to be _told_ to use it but would be made aware by the package itself (believe in the system not the operators), and the default for other models remains the same.
This is how I would implement it:

`base.py`
```python
class ContextualModel(...):
     def print_topics(..., display_negative=False):
          pass
```

`decomp.py`
```python
class SemanticSignalSeparation(ContextualModel):
     def print_topics(..., display_negative=True):
         super().print_topics(..., display_negative=display_negative)
```

**Comment by x-tabdeveloping** [comment]
@rbroc Do you have some code for this? If not I will go ahead and write this cause it is crucial for the case study.

**Comment by rbroc** [comment]
go ahead if high-priority -- my code requires some work but won't be able to look into it until tuesday

**Comment by rbroc** [comment]
closed by #31
